cmake/cores/Cortex-M0+.cmake

```diff
@@ -27,7 +27,7 @@ function(mbed_set_cpu_core_options target mbed_toolchain)
             PUBLIC
                 $<$<COMPILE_LANGUAGE:C>:${compile_options}>
                 $<$<COMPILE_LANGUAGE:CXX>:${compile_options}>
-                $<$<COMPILE_LANGUAGE:ASM>:--cpu=Cortex-M0plus>
+                $<$<$<$<COMPILE_LANGUAGE:ASM>:-mcpu=Cortex-M0plus>
```

Comment:
I'm not a CMake expert but the extra `$<$<` looks imbalanced?

Reply:
Good catch, fixing now